client/state/preferences/actions.js

```diff
@@ -97,10 +90,5 @@ export const savePreference = ( key, value ) => ( dispatch ) => {
 				value,
 			} );
 		} )
-		.catch( ( error ) => {
-			dispatch( {
-				type: PREFERENCES_SAVE_FAILURE,
-				error,
-			} );
-		} );
+		.catch( () => {} );
```

Comment:
I wonder what should happen if that request fails. Ideally, we would show some error notice (not part of this PR I think). Right now, if I block network requests to `/me/preferences` and change the color scheme it still says _Settings saved successfully_.

Reply:
I agree with you both, and indeed the best way is to leave the component to handle the error.

However `savePreference()` is used in plenty of places (20+ instances total) and we'd have to test all of them just for the sake of this cleanup. We had a `catch()` in the previous version of the code and it's possible one or more usages rely on the errors being caught here. 

So, I prefer keeping it as-is for now, and maybe clean up in the future if you folks are feeling strongly about it 😉